Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -553,8 +553,8 @@ void EnhancementsWidget::AddDescriptions()
       "causes slowdowns or graphical issues.<br><br><dolphin_emphasis>If unsure, leave "
       "this unchecked.</dolphin_emphasis>");
   static const char TR_WIDESCREEN_HACK_DESCRIPTION[] = QT_TR_NOOP(
-      "Forces the game to output graphics for any aspect ratio. Use with \"Aspect Ratio\" set to "
-      "\"Force 16:9\" to force 4:3-only games to run at 16:9.<br><br>Rarely produces good "
+      "Forces the game to output graphics for any aspect ratio. (e.g.) Use with \"Aspect Ratio\" "
+      "set to \"Force 16:9\" to force 4:3-only games to run at 16:9.<br><br>Rarely produces good "
```

Comment:
@MayImilae, any comments about the working here? Mainly thinking about the way "e.g." is used.

Reply:
The point here is to highlight that this feature isn't exclusively made for the "Force 16:9" aspect ratio setting, it works with "Stretch" and "Custom" aspect ratios as well.

Reply:
Yes, I don't think it's a bad thing. I just found the way of sticking "e.g." in brackets to be a bit odd.

Reply:
any suggestions?

Reply:
> Forces the game to output graphics for any aspect ratio. (e.g.) Use with "Aspect Ratio" set to "Force 16:9" to force 4:3-only games to run at 16:9.

Honestly I think that example makes things more confusing, rather than helping. Like, the bit that explains it is the start "force a game to output graphics for any aspect ratio" is how it works. If we want to explain applications and usage for it though, we'd have to write a paragraph talking about how it increases FOV and things. Saying it can force 4:3 games to 16:9 isn't wrong but it isn't making it obvious why you'd want to use it. 

I'm also not happy with a few other bits of the language here, such as "for any aspect ratio" (should be at), "rarely produces good results" (it's more 50-50 in my experience, not great but hardly "rare"), and it's focus on widescreen when it can do any aspect ratio from ultrawide to portrait. Let me give you an alternate take for this that may be better.

>Forces the game to output graphics at any aspect ratio without stretching the image by expanding the view frustum. This is a hack, and its results will vary widely game to game. Game-specific AR/Gecko-code aspect ratio patches will out perform this hack, use them instead whenever possible.

Reply:
As someone who doesn't have English as my main language and that will have to deal with translating this later, what a "view frustum" is supposed to mean in that context? I looked up in a few dictionaries but I'm still having a hard time correlating the meaning of that expression with what this setting actually does...

Reply:
It comes up pretty quickly with a Google. https://en.wikipedia.org/wiki/Viewing_frustum

It's the technical term for what is within the pyramid of the camera's field of view. We could avoid the term but it would be a bit wordy to explain without it, plus it's pretty easy to google. It's also where the term "frustum culling" comes from, which is a parlance that is reasonably well known in gaming culture... at least in English native countries.

Reply:
Ok, the new wording is much better, I'll change to that.